### ISSUE TITLE
Create an invoice

### ISSUE THREAD

**Issue by trystant** [trystant/quickbooks-rails-5#1]
I want to know how to create an invoice using this integration

**Comment by trystant** [comment]
I am using this [tutorial](http://minimul.com/integrating-rails-and-quickbooks-online-via-the-version-3-api-part-1.html) by @minimul, but using Rails 5

**Comment by minimul** [comment]
Yeah, that is a very dated tut and I am not going to have time to update it.

**Comment by trystant** [comment]
@minimul Cool, I'm surprised you responded. I am going to do some work for my own edification. If I end up making an updated version of this, would you mind if I link to your blog?  Thanks.

**Comment by austinklenk** [comment]
@trystant I'm trying to integrate qbo with my rails app to lol
